STANDARD:
  axes:
    left_stick_horizontal: 0
    left_stick_vertical: 1
    right_stick_horizontal: 2
    right_stick_vertical: 3
  buttons:
    A_button: 0
    B_button: 1
    X_button: 2
    Y_button: 3
    left_bumper: 4
    right_bumper: 5
    left_trigger: 6
    right_trigger: 7
    back_button: 8
    start_button: 9
    left_stick_button: 10
    right_stick_button: 11
    up_cross: 12
    down_cross: 13
    left_cross: 14
    right_cross: 15
    power_button: 16

XBOX_XS:
  axes:
    left_stick_horizontal: 0
    left_stick_vertical: 1
    right_stick_horizontal: 3
    right_stick_vertical: 4
    left_trigger: 2
    right_trigger: 5
    cross_horizontal: 6
    cross_vertical: 7
  buttons:
    A_button: 0
    B_button: 1
    X_button: 2
    Y_button: 3
    left_bumper: 4
    right_bumper: 5
    back_button: 6
    start_button: 7
    power_button: 8
    left_stick_button: 9
    right_stick_button: 10

XBOX_360:
  axes:
    left_stick_horizontal: 0
    left_stick_vertical: 1
    right_stick_horizontal: 2
    right_stick_vertical: 3
    left_trigger: 4
    right_trigger: 5
    cross_horizontal: 6
    cross_vertical: 7
  buttons:
    A_button: 0
    B_button: 1
    X_button: 2
    Y_button: 3
    left_bumper: 4
    right_bumper: 5
    back_button: 6
    start_button: 7
    power_button: 8
    left_stick_button: 9
    right_stick_button: 10
